src/queue.jl

```diff
@@ -13,11 +19,21 @@ length(s::Queue) = length(s.store)
 front(s::Queue) = front(s.store)
 back(s::Queue) = back(s.store)
 
+"""
+    enqueue!(s::Queue, x)
+
+Inserts the value `x` to the end of the queue `s`.
+"""
 function enqueue!(s::Queue, x)
     push!(s.store, x)
     s
 end
 
+"""
+    dequeue!(s::Queue)
+
+Removes an element from the back of the queue `s`.
```

Comment:
Similarly to how `enqueue!` was flipped, so is this.

`dequeue!` should (and does) remove an item from the front of the queue